contrib/environment-to-ini/environment-to-ini.go

```diff
@@ -143,15 +145,21 @@ func runEnvironmentToIni(c *cli.Context) error {
 				continue
 			}
 		}
+		oldValue := key.Value()
+		if oldValue != value {
```

Comment:
```suggestion
		if !changed && oldValue != value {
```
set changed once ... since changed is boolean it should be faster for go to check and only if changed == false compare values ...

Reply:
Fair enough - This isn't really a hot path but may aswell try to be a little more efficient